docs/about/release-notes.rst

```diff
@@ -15,12 +15,22 @@ Breaking changes
 Bugfixes
 ~~~~~~~~
 
+Deprecations
+~~~~~~~~~~~~
+
+* The deprecated ``events`` property has been removed.
```

Comment:
Please add a link / note on what to use instead.

Reply:
There is no simple replacement. It is a whole collection of tools. I don't think this property was in widespread use, so not many users will have to adapt.

Reply:
ok